src/sentry/audit_log/manager.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
+from typing import Callable, List
+
+import sentry_sdk
+
+from sentry.models.auditlogentry import AuditLogEntry
+
+
+@dataclass
+class AuditLogEvent:
+    event_id: int
+    name: str
+    api_name: str
+    # render holds a function that will determine the message to be displayed in the audit log
+    render: Callable[[AuditLogEntry], str]
```

Comment:
This works for me. We can still create subclasses for more complex audit logs that need more than a lambda to render.